lightwood/data_schemas/predictor_config.py

```diff
@@ -33,6 +37,7 @@
     'output_features': [
         feature_schema
     ],
-    Optional('mixer'): mixer_schema,
+    Optional('data_source', default=data_source_schema.validate({})): data_source_schema,
```

Comment:
lets discuss this, not sure how this is in the config, but would like to understand better your reasoning here

Reply:
cache_encoded_data flag?